docs/register-create-stake-phase-2.md

```diff
@@ -0,0 +1,522 @@
+  provider double signs leading to the exposure of their private key
+  the slashing transaction can be propagated to Bitcoin.
+- **Proof of Possession**: Verifies ownership of the Bitcoin key.
+- **Associated Staking Data**: Includes necessary metadata for the stake.
+- **Merkle Proof of Inclusion** (optional): Provides proof of transaction 
+inclusion in a Bitcoin block.
+
+The associated data needs to be bundled into a Babylon chain transaction
+and broadcasted to the Babylon network depending on the flow to be followed
+(either pre-stake registration or post-stake registration).
+
+The following sections cover:
+- Required parameters for Bitcoin staking transactions.
+- Construction of the Babylon transaction and required staking data.
+- Submission and monitoring of the registration transaction on the Babylon chain.
+
+### 3.2. Babylon Chain BTC Staking Parameters
+
+<!-- TODO: the parameters are also Bitcoin height specific,
+we need to add details about the Bitcoin height parameterization of
+these params
+Need to catch-up with someone from @core for this.-->
+
+Bitcoin Staking transactions need to abide by the parameters specified by the
+Babylon chain for staking transaction validity. Below, we provide
+a brief overview of the parameters employed by the Babylon chain
+as part of the [x/btcstaking](../x/btcstaking) Cosmos SDK module:
+
+* `covenant_pks`:
+  A list containing of the BIP-340 public keys used by the covenant committee. 
+  Expressed as a 64-character hex string. The pk is an x-coordinate only 
+  representation of a secp256k1 curve point as the y-coordinate is implied.
+* `covenant_quorum`:
+  The minimum number of signatures to achieve a quorum from the
+  covenant committee in uint32 format.
+* `min_staking_value_sat`:
+  The minimum amount of statoshis (smallest unit of Bitcoin) required to be 
+  locked in the staking output (i.e., staked) in int64 format in order for the 
+  staking transaction to be valid.
+* `max_staking_value_sat`:
+  The maximum amount of statoshis (smallest unit of Bitcoin) required to be 
+  locked in the staking output (i.e., staked) in int64 format in order for the 
+  staking transaction to be valid.
+  (i.e., staked) in int64 format in order for the staking transaction to be valid.
+* `min_staking_time_blocks`:
+  The minimum staking time as the number of Bitcoin blocks. This number
+  should be added in the locktime specified in the staking output script.
+* `max_staking_time_blocks`:
+  The maximum staking time as the number of Bitcoin blocks. This number
+  should be added in the locktime specified in the staking output script.
+* `slashing_pk_script`:
+  The `pk_script` expected in the slashing output, i.e., the first output of 
+  the slashing transaction. It is stored as a sequence of bytes, representing 
+  the conditions for spending the output.
+* `min_slashing_tx_fee_sat`:
+  The minimum transaction fee (in satoshis) required for the pre-signed slashing
+  transaction.
```

Comment:
I would also add to the list:
- `unbonding_time_blocks` - required time lock on unbonding transactions (and change output of slashing transactions)
- `unbonding_fee_sat` - fee required for unbonding transactions

Imo it is important that those two are here, as they impact the unbonding transaction